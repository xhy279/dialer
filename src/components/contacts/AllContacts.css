html, body, #root  {
  height: 99%;
  background-color: #f5f5f5;
}
.all-contacts {
  height: 100%;
}

.all-contacts-bar {
  position: sticky;
  top: 0;
  left: 0;
  z-index: 999;
}
.all-contacts__content {
  height: 80%;
}
.all-contacts__content-loader {
  position: absolute;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
}