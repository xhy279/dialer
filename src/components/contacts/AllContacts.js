import React from 'react';
import CircularProgress from '@material-ui/core/CircularProgress';
import ContactList from './ContactList';
import Header from '../header/Header';
import './AllContacts.css';

const AllContacts = ({users}) => {
  return (
    <div className="all-contacts">
      <Header/>
      <div className="all-contacts__content">
        {users.length > 0
          ? <ContactList users={users}/>
          : <div className="all-contacts__content-loader">
            <CircularProgress/>
          </div>
}

      </div>
    </div>
  );
}

export default AllContacts;