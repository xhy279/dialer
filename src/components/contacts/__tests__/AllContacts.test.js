import { shallow } from 'enzyme';
import React from 'react';
import AllContacts from '../AllContacts';

it( 'expect to match snapshot', () => {
  expect(shallow(<AllContacts />)).toMatchSnapshot();
})