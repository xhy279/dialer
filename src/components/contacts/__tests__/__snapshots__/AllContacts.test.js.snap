// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to match snapshot 1`] = `ShallowWrapper {}`;
